persistent/interfaces.py

```diff
@@ -23,7 +23,7 @@
     from persistent.cPersistence import UPTODATE
     from persistent.cPersistence import CHANGED
     from persistent.cPersistence import STICKY
-except ImportError: #pragma NO COVER
+except ImportError: # pragma: no cover
```

Comment:
OK. The whitespace cleanup was an automatic result of the spacing cleanup on this line (I contribute to some projects (gevent) that are *really* picky about trailing whitespace).

I don't get complete coverage, though:
```
$ tox -e coverage
GLOB sdist-make: //persistent/setup.py
coverage create: //persistent/.tox/coverage
coverage installdeps: zope.interface, nose, coverage, nosexcover, cffi
coverage inst: /persistent/.tox/dist/persistent-4.0.10.dev0.zip
coverage installed: -f file:////buildout-eggs,cffi==0.9.2,coverage==3.7.1,nose==1.3.6,nosexcover==1.0.10,persistent==4.0.10.dev0,pycparser==2.13,zope.interface==4.1.2
coverage runtests: PYTHONHASHSEED='30752747'
coverage runtests: commands[0] | nosetests --with-xunit --with-xcoverage
.....................................................................................................................................................................................................................................................................................................................................
Name                     Stmts   Miss Branch BrMiss  Cover   Missing
--------------------------------------------------------------------
persistent                  21     11      0      0    48%   24-46
persistent._compat           1      0      0      0   100%
persistent.dict              1      0      0      0   100%
persistent.interfaces        9      3      0      0    67%   23-25
persistent.list             60      0      0      0   100%
persistent.mapping          44      0      4      0   100%
persistent.persistence     318      0    155      0   100%
persistent.picklecache     199      0     96      0   100%
persistent.ring             81      0     18      0   100%
persistent.timestamp       125      0     14      0   100%
persistent.timestamp       125      0     14      0   100%
persistent.wref             68      0     18      0   100%
--------------------------------------------------------------------
TOTAL                     1052     14    319      0    99%
nose.plugins.cover: ERROR: TOTAL Coverage did not reach minimum required: 100%
ERROR: InvocationError: '//persistent/.tox/coverage/bin/nosetests --with-xunit --with-xcoverage'
___________________________________________________________________________________________________________ summary ____________________________________________________________________________________________________________
ERROR:   coverage: commands failed
$ .tox/coverage/bin/python
Python 2.6.9 (unknown, Sep  9 2014, 15:05:12)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> ^D
```